docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -18,11 +18,27 @@ YugabyteDB Anywhere allows you to monitor and troubleshoot issues that arise fro
 
 ## Use metrics
 
-A universe's **Metrics** page displays graphs representing information on operations, latency, and other parameters accumulated over time for each type of node and server. Examining specific metrics allows you to diagnose and troubleshoot issues.
+A universe's **Metrics** page displays charts representing information on operations, latency, and other parameters accumulated over time for each type of node and server. Examining specific metrics allows you to diagnose and troubleshoot issues.
 
 You access metrics by navigating to **Universes > Universe-Name > Metrics**, as shown in the following illustration:
 
-![Metrics Page](/images/yp/metrics.png)<br>
+![Metrics Page](/images/yp/metrics_main.png)<br>
+
+You can chose to view metrics based on different criteria:
+
+- If your universe has more than one region, cluster, zone, or node, the **Metrics** page allows you to view metrics for a specific region, cluster, zone, or node.
+
+- You can view the overall node metrics.
```

Comment:
We should somehow say that we show metrics, aggregated across the selected nodes here. And that we show either average across nodes, or sum of the all node values depending on the metric type.

Reply:
Added